src/main/java/mekanism/common/integration/OreDictManager.java

```diff
@@ -93,17 +93,20 @@ public static void init(RegistryEvent.Register<IRecipe> event) {
         }
 
         for (ItemStack sulfur : OreDictionary.getOres("dustSulfur", false)) {
-            RecipeHandler.addChemicalOxidizerRecipe(StackUtils.size(sulfur, 1), new GasStack(MekanismFluids.SulfurDioxide, 100));
+            sulfur = StackUtils.size(sulfur, 1);
+            RecipeHandler.addChemicalOxidizerRecipe(sulfur, new GasStack(MekanismFluids.SulfurDioxide, 100));
+            RecipeHandler.addEnrichmentChamberRecipe(sulfur, new ItemStack(Items.GUNPOWDER));
         }
 
         for (ItemStack salt : OreDictionary.getOres("dustSalt", false)) {
             RecipeHandler.addChemicalOxidizerRecipe(StackUtils.size(salt, 1), new GasStack(MekanismFluids.Brine, 15));
         }
 
         for (ItemStack dust : OreDictionary.getOres("dustRefinedObsidian", false)) {
-            RecipeHandler.addOsmiumCompressorRecipe(StackUtils.size(dust, 1), new ItemStack(MekanismItems.Ingot, 1, 0));
-            RecipeHandler.addEnrichmentChamberRecipe(StackUtils.size(dust, 1), new ItemStack(MekanismItems.CompressedObsidian));
-            InfuseRegistry.registerInfuseObject(StackUtils.size(dust, 1), new InfuseObject(InfuseRegistry.get("OBSIDIAN"), 10));
+            dust = StackUtils.size(dust, 1);
```

Comment:
Is it ok like this or should i add a new ItemStack var for the ItemStack StackUtils.size returns?

Reply:
Its fine like this I think, don't stress too much on the minor details, just what you think is "best"

Reply:
ok than i finished this i guess